### PR TITLE
Update Poetry version in CI

### PR DESCRIPTION
**Pull request by sujuka99**

### CODE REVIEW

File: .github/actions/update-docs/action.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ runs:
     - name: Install python and set up Poetry
```
Comment (disrupted) [inline .github/actions/update-docs/action.yml:24]: please capitalize Python

Comment (sujuka99) [reply .github/actions/update-docs/action.yml]: done

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Install python and set up Poetry
```
Comment (disrupted) [inline .github/workflows/ci.yaml:24]: here as well

**Review by disrupted — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (sujuka99) [reply .github/workflows/ci.yaml]: done

**Review by disrupted — APPROVED** [review]

**Pull request merged**